RevenueCatUI/Views/GradientView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  GradientView.swift
+//
+//  Created by Mark Villacampa on 2024-11-24.
+
+import SwiftUI
+
+#if PAYWALL_COMPONENTS
+
+struct GradientView: View {
+    enum GradientStyle {
+        case linear(Int)
+        case radial
+    }
+
+    @Environment(\.colorScheme)
+    private var colorScheme
```

Comment:
For completeness, and in case we want to use use `GradientView` somewhere else,  I've kept support for light/dark mode here even though we're controlling it in `BackgroundStyle` and `ForegroundColorScheme`.